exercises/concept/wizards-and-warriors/.docs/instructions.md

```diff
@@ -16,19 +16,39 @@ For a Wizard, these are the rules:
 
 In general, fighters are never vulnerable. However, Wizards _are_ vulnerable if they haven't prepared a spell.
 
-You have six tasks that work with Warriors and Wizard fighters.
+You have eight tasks that work with Warriors and Wizard fighters.
 
-## 1. Describe a fighter
+## 1. Create the Warrior and Wizard classes
 
-Override the `toString()` method on the `Fighter` class to return a description of the fighter, formatted as `"Fighter is a <FIGHTER_TYPE>"`.
+Define the classes `Wizard` and `Warrior` outside of the `Fighter` class
+
+```java
+public class Warrior {
+
+}
+```
+
+## 2. Extend Wizard and Warrior classes with Figther class
+
+Use the `extends` keyword to inherit the `Fighter` class.
+
+```java
+public class Warrior extends Fighter {
+    
+}
+```
```

Comment:
This instruction is a bit too verbose, it literally shows what to type into your editor. I suggest instructing _what_ to do, not _how_ to do it.

We want the student to make the `Warrior` and `Wizard` classes to inherit from the `Fighter` class.

Reply:
Done!

Reply:
I think we should not include the code snippets either

Reply:
Oh I thought we should always add them, I will remove them then